pkg/capacity/manager_test.go

```diff
@@ -145,11 +145,11 @@ func TestGetOptimalTargetCluster(t *testing.T) {
 		assert.Equal(t, "member2", clusterName)
 	})
 
-	t.Run("with two clusters, none of them is returned since it reaches max number of MURs, no matter what is defined as preferred", func(t *testing.T) {
+	t.Run("with two clusters, none of them is returned since it reaches max number of Spaces, no matter what is defined as preferred", func(t *testing.T) {
 		// given
 		toolchainConfig := commonconfig.NewToolchainConfigObjWithReset(t,
-			testconfig.AutomaticApproval().
-				MaxNumberOfUsers(800, testconfig.PerMemberCluster("member1", 6000), testconfig.PerMemberCluster("member2", 1000)).
+			testconfig.CapacityThresholds().
+				MaxNumberOfSpaces(testconfig.PerMemberCluster("member1", 6), testconfig.PerMemberCluster("member2", 10)).
 				ResourceCapacityThreshold(80, testconfig.PerMemberCluster("member1", 60), testconfig.PerMemberCluster("member2", 75)))
 		fakeClient := NewFakeClient(t, toolchainStatus, toolchainConfig)
 		InitializeCounters(t, toolchainStatus)
```

Comment:
If I'm not mistaken, this one was testing the "overall" limit which was dropped, so I guess that we can drop also this test.

Reply:
right, but now is covering the case in which there is no more room for spaces in both members (even if the limit is per member cluster and not overall anymore).

Do you think we don't need to test that?